src/index.html

```diff
@@ -729,10 +729,13 @@ <h5>Acknowledgement</h5>
 
                                             <b>Is your data represented in a valid VCF file?</b>
                                             <p>
-                                                Genetic variants submitted to the EVA must be described in the <a href="http://samtools.github.io/hts-specs/" target="_blank">Variant Call Format (VCF)</a>. 
-                                                The EVA only accepts VCF files that adhere to the specification and match a reference sequence. 
+                                                Genetic variants submitted to the EVA must be described in the <a href="https://samtools.github.io/hts-specs/VCFv4.3.pdf" target="_blank">Variant Call Format (VCF)</a>.
+                                                The EVA only accepts VCF files that adhere to the specification and match a reference sequence (Versions <a href="https://samtools.github.io/hts-specs/VCFv4.1.pdf" target="_blank">v4.1</a>, <a href="https://samtools.github.io/hts-specs/VCFv4.2.pdf" target="_blank">v4.2</a>, & <a href="https://samtools.github.io/hts-specs/VCFv4.3.pdf" target="_blank">v4.3</a> accepted)
                                                 Submitters can make use of the <a href="https://github.com/EBIvariation/vcf-validator/wiki/User's-Guide" target="_blank">EVA VCF validation suite</a> to ensure they fulfill these requirements. 
                                                 Furthermore, VCF files submitted to the EVA should provide either sample genotypes and/or aggregated sample summary-level allele frequencies.
+                                                Variant lines should also be unique and not specify duplicate loci as this causes issues during steps such as calculation of population frequencies.
+                                                Although both are accepted, single VCF files containing sample information through the FORMAT field are preferred over individual VCF files for each sample.

```

Comment:
To be easy on non-native readers:
```suggestion
                                                Although both per-sample VCFs and merged VCFs are acceptable, merged VCF files containing sample information through the FORMAT field are preferred over separate VCF files for each sample. Single VCF files allow data to be easily reusable and reproducible. Therefore, we recommend merging per-sample VCF files before submission. 
```